eng/scripts/ChangedFiles-Functions.ps1

```diff
@@ -0,0 +1,56 @@
+function Get-ChangedFiles($baseCommitsh="HEAD^", $targetCommitish="HEAD", $diffFilter="d")
```

Comment:
I think the functions in here are all vulnerable to the powershell single item casting gotcha, possibly you want to [array] cast all the assigned variables in the other scripts?

Reply:
Just so I'm clear you are suggesting the callers do `@(Get-ChangedFiles)`? I believe most of the current usages should be fine but you are right new usages could get hit.